base/services/w32time/ntpclient.c

```diff
@@ -24,15 +22,13 @@ typedef struct _INFO
 
 
 static BOOL
-InitConnection(PINFO pInfo,
-               LPSTR lpAddress)
+InitConnection(PINFO pInfo, LPSTR lpAddress)
```

Comment:
Now that the diff shows actual changes here, I'd argue we should undo most of what's in this file, since it's just changing the formatting of code that doesn't actually need to be changed. Make sense?

Reply:
I agree.